TrackingProduction/Fun4All_TrkrHitSet_Unpacker.C

```diff
@@ -80,46 +76,8 @@ void Fun4All_TrkrHitSet_Unpacker(
   tpotunpacker->set_calibration_file(calibrationFile);
   se->registerSubsystem(tpotunpacker);
 
-  Mvtx_Clustering();
-  Intt_Clustering();
-
-  auto tpcclusterizer = new TpcClusterizer;
-  tpcclusterizer->Verbosity(0);
-  tpcclusterizer->set_do_hit_association( G4TPC::DO_HIT_ASSOCIATION );  
-  tpcclusterizer->set_rawdata_reco();  
-  se->registerSubsystem(tpcclusterizer);
- 
-  Micromegas_Clustering();
-
-  Tracking_Reco_TrackSeed();
-  convert_seeds();
-
-  TString residoutfile = outfilename + runnumber + "_resid.root";
-  std::string residstring(residoutfile.Data());
-  
-  auto resid = new TrackResiduals("TrackResiduals");
-  resid->outfileName(residstring);
-  resid->alignment(false);
-  se->registerSubsystem(resid);
-
-  TString ntupoutfile = outfilename + runnumber + "_ntup.root";
-  std::string ntupstring(ntupoutfile.Data());
-
-  auto ntp = new TrkrNtuplizer("TrkrNtuplizer",ntupstring);
-  ntp->do_hit_eval(true);
-  ntp->do_cluster_eval(true);
-  ntp->do_track_eval(true);
-  ntp->do_siseed_eval(true);
-  ntp->do_tpcseed_eval(true);
-  ntp->do_clus_trk_eval(true);
-  ntp->do_vertex_eval(false);
-  ntp->set_trkclus_seed_container("SvtxTrackSeedContainer");
-  ntp->Verbosity(0);
-  ntp->do_info_eval(false);
-  se->registerSubsystem(ntp);
-
-  //Fun4AllOutputManager *out = new Fun4AllDstOutputManager("out", "/sphenix/tg/tg01/hf/jdosbo/tracking_development/onlineoffline/hitsets.root");
-  //se->registerOutputManager(out);
+  Fun4AllOutputManager *out = new Fun4AllDstOutputManager("DSTOUT", outfilename);
```

Comment:
We probably need to strip out the RawHit DST nodes here so the output is compact

Reply:
[Commit](https://github.com/sPHENIX-Collaboration/macros/pull/721/commits/19128021952810c4a948d17830713dc754d4047c) that strips all nodes except the hitset node